build-scripts/post_build.sh

```diff
@@ -32,6 +48,7 @@ CWLAIRFLOW_VERSION="1.2.6"
 CWLAIRFLOW_PYTHON_VERSION="3.8"
 CWLAIRFLOW_MACOS_VERSION="10.15.7"
 BIOWARDROBE_NG_VERSION="2.0.0"
+BIOWARDROBE_NG_LOCAL_PATH=""        # if not "", this absolute path will be used and BIOWARDROBE_NG_VERSION will be ignored 
```

Comment:
Better assign to env variable or get it from .env in order do not commit changes in .sh file